src/main/resources/db/migration/V1_4__modify_idam_id_datatype_to_varchar.sql

```diff
@@ -0,0 +1,3 @@
+alter table user_profile drop column idam_id;
```

Comment:
DO NOT UNDER ANY CIRCUMSTANCES DROP THIS COLUMN. You can alter the table instead.
ALTER TABLE user_profile ALTER COLUMN idam_id TYPE VARCHAR;

Reply:
ok.